docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -274,12 +253,19 @@ spec:
       openstack:
         secret: osp-secret
         replicas: 3
-        storageRequest: 500M
+        storageRequest: 5G
       openstack-cell1:
         secret: osp-secret
         replicas: 3
-        storageRequest: 500M
-
+        storageRequest: 5G
+      openstack-cell2:
+        secret: osp-secret
+        replicas: 1
+        storageRequest: 5G
+      openstack-cell3:
+        secret: osp-secret
+        replicas: 1
+        storageRequest: 5G
```

Comment:
Do we need to make it conditional so that the user only creates it if there is cell2 in the source deployment?

Reply:
I noted in this quide that it creates a static deployment topology. This is needed to match tests with docs.
Making it (both!) fully dynamic is not something we can do today.